framework/src/test/java/org/tron/core/db/AccountStoreTest.java

```diff
@@ -61,6 +70,21 @@ public void init() {
     init = true;
   }
 
+  @Test
+  public void setAccountTest() throws Exception {
+    Field field = AccountStore.class.getDeclaredField("assertsAddress");
+    field.setAccessible(true);
+    field.set(AccountStore.class, new HashMap<>());
+    Config config = mock(Config.class);
+    Mockito.when(config.getObjectList("genesis.block.assets")).thenReturn(new ArrayList<>());
+    try {
+      AccountStore.setAccount(config);
+      Assert.fail();
+    } catch (Throwable e) {
+      Assert.assertTrue(e instanceof TronError);
```

Comment:
add `Assert.assertEquals("Account[Blackhole] is not configured.", e.getMessage());`?